.github/ISSUE_TEMPLATE/fix-content.yml

```diff
@@ -0,0 +1,44 @@
+name: Fix Content
+description: Report issues with the documentation content
+labels: ['update']
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to submit this issue. Your input is appreciated.
+  - type: textarea
+    id: issue-location
+    attributes:
+      label: Where is the issue?
+      description: Link to the documentation page(s) where you identified this issue.
+      placeholder: 
+  - type: checkboxes
+    id: issue-type
+    attributes:
+      label: Issue type
+      description: Select the type(s) of issue you are reporting.
+      options:
+        - label: Missing content
+          required: false
+        - label: Outdated content
+          required: false
+        - label: Inaccurate information
+          required: false
+  - type: textarea
+    id: issue-description
+    attributes:
+      label: Detailed issue description
+      description: Describe the issue in more detail.
+      placeholder: What needs to change?
+      required: true
+  - type: textarea
+    id: text-resolution
+    attributes:
+      label: What should be in the docs?
+      description: Provide a suggested resolution if available.
+      placeholder:
+  - type: textarea
+    id: information
+    attributes:
+      label: Additional information
```

Comment:
Having three sections to describe the issue details is overkill. As a frequent issue raiser, especially for content fixes I'd rather not see all these options and just write a single straightforward description.

Reply:
My main issue is that these templates have too many sections with overlapping and synonymous prompts that would frustrate me as an issue raiser. Can we compromise by trying two text boxes each for the "Fix content" and "New content" templates? One to describe the issue in detail, and one to provide a suggested change.

Reply:
I've been thinking - having just two text boxes might not fully help. It's not about asking for lots of text from those opening issues, but more about giving a bit of direction to frame their context. The text fields are optional, especially since our docs team (especially you and Byron) handle the majority of the issue opening. I get that it might feel cumbersome to answer several prompts. However, for projects involving community engagement, a little more info upfront can be really useful. In my limited experience (over the last month), I often find myself asking follow-up questions that could have been covered initially with just a few more details. This isn't about making big changes; it's about trying something new to see if it helps. I think we can give it a go and see how it goes. We can always tweak it, make it more concise, or even drop it entirely if it turns out to be more of a hindrance than a help.

Reply:
From our meeting: Let's remove the "additional information" text area and make the labels, descriptions, and placeholders of the other text areas more distinct and consistent.